docs/library/network.rst

```diff
@@ -210,6 +210,31 @@ For example::
 
         Disconnect from the currently connected wireless network.
 
+    .. method:: wlan.scan(cb)
+
+        Initiate scanning for the available wireless networks.
+
+        Scanning is only possible if the radio is in station or station+AP mode; if
+        called while in AP only mode, an exception will be raised.
+
+        Once the scanning is complete, the provided callback function ``cb`` will
+        be called once for each network found, and passed a tuple with information
+        about that network:
+
+            (ssid, bssid, channel, RSSI, authmode, hidden)
+
+        There are five values for authmode:
+
+            * 0 -- open
+            * 1 -- WEP
+            * 2 -- WPA-PSK
+            * 3 -- WPA2-PSK
+            * 4 -- WPA/WPA2-PSK
+
+        and two for hidden:
```

Comment:
I think we should change the hidden parameter so it returns False or True, then it doesn't need documenting like this.